site/docs/actions/public/L1/readProvenWithdrawals.md

```diff
@@ -0,0 +1,42 @@
+# readProvenWithdrawals
+
+Returns a `ProvenWithdrawal` struct containing the `outputRoot`, `timestamp`, and `l2OutputIndex` for a given withdrawal hash. Returns error if withdrawal has not been proven.
```

Comment:
We should make a TODO to put ProvenWithdrawal in types/glossary so we can point to it from multiple places

Reply:
Working on that in #115